views/github-create-branch.hbs

```diff
@@ -48,7 +48,7 @@
       <aui-label for="repo">Repository</aui-label>
       <select id="ghRepo" name="repo" class="full-width-field" placeholder="Select a repository">
         {{#each repos}}
-          <option value="{{id}}">{{org}} / {{name}}</option>
+          <option value="{{node.id}}">{{node.owner.login}} / {{node.name}}</option>
```

Comment:
We need `orgName` and `repo` for creating branch. will it make sense to put this information in value as well?  We're not using id anywhere.

Reply:
Replaced the `id` by `full_name`(which is the same value as `owner.login / name`)